agent/test/filemover/FileMover.java

```diff
@@ -0,0 +1,33 @@
+import java.io.IOException;
```

Comment:
It doesn't look like this test checks anything that's not also checked by `RecorderTest`? If that's true, I'd say it can be removed.

Reply:
This test checks the `AtomicMoveNotSupportedException` exception doesn't lead to `Recording` printing `Invalid cross-device link`. `RecorderTest` wasn't checking this.

Reply:
Ok. That's kind of minor payoff for quite a bit of code. Seems like you could get the same result in RecorderTest by adding a test that captures `stderr`, yes? You can see a similar test [here](https://github.com/getappmap/appmap-java/blob/02adad7a8706137a137cd9940982143051f44368/agent/src/test/java/com/appland/appmap/test/util/ClassBuilderTest.java#L5), though you'll want to use `org.junit.contrib.java.lang.system.SystemErrRule` rather than `SystemOutRule`.

Reply:
Thanks, changed to use `SystemErrRule` and removed `filemover` integration test.